dnf5daemon-server/services/advisory/advisory.cpp

```diff
@@ -73,7 +74,7 @@ libdnf5::advisory::AdvisoryQuery Advisory::advisory_query_from_options(
         // Filter by advisory severity
         if (!opt_severities.empty()) {
             auto advisories_severities = libdnf5::advisory::AdvisoryQuery(base);
-            advisories_severities.filter_severity(opt_severities);
+            advisories_severities.filter_severity(opt_severities, libdnf5::sack::QueryCmp::IEXACT);
```

Comment:
This fixes the first letter capitalization requirement, however, it is now a bit inconsistent that the "severities" are case insensitive, while "availability" and "types" remain case sensitive. My question is if it shouldn't be all case insensitive. But it's just a minor thing.

Reply:
Thinking about it once more - what do you think about setting `libdnf5::sack::QueryCmp::IEXACT` as the default comparator directly in `include/libdnf5/advisory/advisory_query.hpp`? Could be done also for types.
That would fix also future potential issues with case sensitivity for API users.

Reply:
I think it would make sense for those filters that accept values from predefined lists, which is the case for severities and types.

Reply:
I changed default comparator in filter_severity and filter_type to IEXACT. Also availability is now handled as case insensitive.

Reply:
Thank you!